whitepapers/business-value/Introduction.md

```diff
@@ -0,0 +1,119 @@
+# What is open source? 
+
+When it comes to open source, there are a lot of misconceptions – even among software engineers. So what exactly is open source? 
+
+Open source software has its roots in the free software movement, which started in the 1970s. Software was the first ‘digital product.’ Long before there were digital music or digital books, you could create a copy of a software program for free, without any impact on the original program. This gave rise to a political philosophy that holds that programmers shouldn’t charge for software, and that software should be a source of empowerment for users. The ‘free’ in the free software movement is a reference to freedom, not to something that costs $0. There are four basic freedoms of the free software movement: 
```

Comment:
"you could create a copy of a software program for free" -> not sure whether free is here meant as in Free Software or as in there were no cost to technically replicate/copy the source code, e.g., for distribution. My initial guess would be the later but maybe one can make this more clear here. It is is an interesting point that for digital goods we miss the additional effort and connected cost for replicating something even when the good is considered a commodity.  This lack of production cost often makes people less willing to pay for digital goods compared to physical items.